inst/Upgrades.qml

```diff
@@ -0,0 +1,342 @@
+		ChangeRename { from: "seedBox";									to: "setSeed"						}
+		ChangeRename { from: "modelOpt";								to: "modelOptimization"				}
+
+		ChangeRename { from: "plotLegend";								to: "legendShown"					}
+		ChangeRename { from: "plotPoints";								to: "pointsShown"					}
+
+		ChangeRename { from: "tableVariableImportance";					to: "variableImportanceTable"		}
+
+		ChangeRename { from: "plotTreesVsModelError";					to: "treesVsModelErrorPlot"			}
+		ChangeRename { from: "plotDecreaseAccuracy";					to: "accuracyDecreasePlot"			}
+		ChangeRename { from: "plotIncreasePurity";						to: "purityIncreasePlot"			}
+
+
+	}
+
+	Upgrade
+	{
+
+		functionName:		"mlRegressionKnn"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
+
+		ChangeRename { from: "scaleEqualSD";							to: "scaleVariables"				}
+		ChangeRename { from: "seedBox";									to: "setSeed"						}
+		ChangeRename { from: "modelOpt";								to: "modelOptimization"				}
+
+		ChangeRename { from: "plotWeights";								to: "weightsPlot"					}
+		ChangeRename { from: "plotErrorVsK";							to: "errorVsKPlot"					}
+		ChangeRename { from: "maxK";									to: "maxNearestNeighbors"			}
+
+	}
+
+	Upgrade
+	{
+
+		functionName:		"mlClassificationKnn"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
+
+		ChangeRename { from: "scaleEqualSD";							to: "scaleVariables"				}
+		ChangeRename { from: "seedBox";									to: "setSeed"						}
+		ChangeRename { from: "modelOpt";								to: "modelOptimization"				}
+
+		ChangeRename { from: "plotLegend";								to: "legendShown"					}
+		ChangeRename { from: "plotPoints";								to: "pointsShown"					}
+		ChangeRename { from: "nSplit";									to: "minObservationsForSplit"		}
+
+		ChangeRename { from: "plotWeights";								to: "weightsPlot"					}
+		ChangeRename { from: "plotErrorVsK";							to: "errorVsKPlot"					}
+		ChangeRename { from: "maxK";									to: "maxNearestNeighbors"			}
+
+
+	}
+
+	Upgrade
+	{
+
+		functionName:		"mlRegressionNeuralNetwork"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
+
+		ChangeRename { from: "scaleEqualSD";							to: "scaleVariables"				}
+		ChangeRename { from: "seedBox";									to: "setSeed"						}
+		ChangeRename { from: "modelOpt";								to: "modelOptimization"				}
+
+		ChangeRename { from: "plotError";								to: "meanSquaredErrorPlot"			}
+		ChangeRename { from: "actFuncPlot";								to: "activationFunctionPlot"		}
+		ChangeRename { from: "errfct";									to: "lossFunction"					}
+		ChangeRename { from: "stepMax";									to: "maxTrainingRepetitions"		}
+		ChangeRename { from: "genSize";									to: "populationSize"				}
+		ChangeRename { from: "maxGen";									to: "maxGenerations"				}
+		ChangeRename { from: "elitismProp";								to: "elitismProportion"				}
+
+		ChangeJS
+		{
+			name: "lossFunction"
+			jsFunction: function(options)
+			{
+				switch options["lossFunction"]
+				{
+					case "sse"		return "sumOfSquares";
+					case "ce"		return "crossEntropy";
+				}
```

Comment:
I don't know whether js allows this, but it seems to me the canonically correct syntax would be
```suggestion
				switch (options["lossFunction"])
				{
					case "sse":		return "sumOfSquares";
					case "ce":		return "crossEntropy";
				}
```

Reply:
for the other upgrades as well